### PR TITLE
Fix to use non-neutron test properly OpenStack clients.

### PR DESCRIPTION
**Pull request by rpromyshlennikov**

### CODE REVIEW

File: mos_tests/conftest.py [hunk 1]
```diff
@@ -167,19 +167,31 @@ def set_openstack_environ(fuel_master_ip):
...
+    wait(os_connection.is_nova_ready,
          timeout_seconds=60 * 5,
          expected_exceptions=Exception,
          waiting_for="OpenStack nova computes is ready")
     logger.info("OpenStack is ready")
-    return os_conn
+    return os_connection
+
+
+@pytest.fixture(scope='class')
```
Comment (gdyuldin) [inline mos_tests/conftest.py:183]: In this case you need to change a bit cleanup fixture logic (not revert env if test is undestructive).

File: mos_tests/environment/os_actions.py [hunk 2]
```diff
@@ -84,6 +85,14 @@ def __init__(self, controller_ip, user='admin', password='admin',
         self.glance = GlanceClient(endpoint=glance_endpoint,
                                    token=token,
                                    cacert=path_to_cert)
+
+        heat_endpoint = self.keystone.service_catalog.url_for(
```
Comment (gdyuldin) [inline mos_tests/environment/os_actions.py:89]: Is heat is mandatory component of OpenStack installation?

Comment (rpromyshlennikov) [inline mos_tests/environment/os_actions.py:89]: Yep, heat is presented in all installations.

File: mos_tests/conftest.py (hunk 1 above)
Comment (rpromyshlennikov) [inline mos_tests/conftest.py:183]: I'm going to fix it.

File: mos_tests/environment/os_actions.py (hunk 2 above)
Comment (TimurNurlygayanov) [inline mos_tests/environment/os_actions.py:89]: It is not mandatory, but it is deploying by default and there is no way to disable it.

**Pull request merged**